testing/ejabberd/APKBUILD

```diff
@@ -1,15 +1,32 @@
 # Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
 # Contributor: Francesco Colista <fcolista@alpinelinux.org>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
 # Maintainer: John Regan <john@jrjrtech.com>
 pkgname=ejabberd
-pkgver=17.09
-pkgrel=3
+pkgver=18.03
```

Comment:
btw https://github.com/processone/ejabberd/releases/tag/18.12.1 released

Reply:
Thanks for the hint, I will upgrade to 18.12.1 in a separate PR after this PR gets merged.